src/components/Vault/BasicVault/VrtCard.tsx

```diff
@@ -0,0 +1,145 @@
+        userPendingRewardTemp,
+        userVrtAllowanceTemp,
+      ] = await Promise.all([
+        vrtTokenContract.methods.balanceOf(account).call(),
+        vrtVaultProxyContract.methods.userInfo(account).call(),
+        vrtVaultProxyContract.methods.getAccruedInterest(account).call(),
+        vrtTokenContract.methods.allowance(account, getVrtVaultProxyAddress()).call(),
+      ]);
+    }
+
+    if (isMounted) {
+      // total info
+      setDailyEmission(
+        new BigNumber(interestRatePerBlockTemp)
+          .div(1e18)
+          .multipliedBy(BLOCK_PER_DAY)
+          .multipliedBy(vaultVrtBalance)
+          .div(1e18)
+          .dp(2, 1),
+      );
+      setInterestRatePerBlock(new BigNumber(interestRatePerBlockTemp));
+      setVaultVrtBalance(new BigNumber(vaultVrtBalanceTemp));
+      setUserVrtBalance(new BigNumber(userVrtBalanceTemp));
+      setUserVrtStakedAmount(new BigNumber(userVrtStakedAmountTemp));
+      setUserPendingReward(new BigNumber(userPendingRewardTemp));
+      setUserVrtAllowance(new BigNumber(userVrtAllowanceTemp));
+    }
+
+    return () => {
+      isMounted = false;
+    };
+  }, [fastRefresh, account]);
+
+  return (
+    <VaultCardWrapper>
+      <CardHeader
+        stakedToken="VRT"
+        rewardToken="VRT"
+        apy={interestRatePerBlock.multipliedBy(DAYS_OF_YEAR * BLOCK_PER_DAY).div(1e18).multipliedBy(100).toFixed(2)}
+        totalStakedAmount={vaultVrtBalance.div(1e18)}
+        totalPendingRewards={vaultVrtBalance.div(1e18)}
+        dailyEmission={dailyEmission}
+        onExpand={() => {
+          setExpanded(!expanded);
+        }}
+      />
+      <div className="content-container">
+        {expanded ? (
+          <CardContent
+            userPendingReward={userPendingReward}
+            userStakedTokenBalance={userVrtBalance}
+            userStakedAllowance={userVrtAllowance}
+            userStakedAmount={userVrtStakedAmount}
+            stakedToken="VRT"
+            rewardToken="VRT"
+            onClaimReward={async () => vrtVaultProxyContract.methods.claim().send({ from: account })}
+            onStake={async (stakeAmount) => vrtVaultProxyContract.methods
+              .deposit(
+                stakeAmount.toFixed(0),
+              )
+              .send({ from: account })}
+            onApprove={async (amt) => vrtTokenContract.methods
+              .approve(
+                vrtVaultProxyContract.options.address,
+                amt.toFixed(10),
+              )
+              .send({ from: account })}
+            onWithdraw={async (amt) => vrtVaultProxyContract.methods
+              .withdraw(
+                amt.toFixed(0),
+              )
+              .send({ from: account })}
+          />
+        ) : null}
+      </div>
+    </VaultCardWrapper>
+  );
+}
+
+const mapStateToProps = ({ account }: State) => ({
+  settings: account.setting,
+});
+
+export default connect(mapStateToProps)(VaultCard);
```

Comment:
Since we don't use `settings`, we can get rid of all that completely and export default component directly:
```suggestion
export default VaultCard;
```

Reply:
yep